lib/Agrammon/Documentation.pm6

```diff
@@ -0,0 +1,111 @@
+use Agrammon::Model;
+
+#| Create LaTeX document of Model
+sub prepare-model( Agrammon::Model $model, :%technical! ) is export {
+
+    my @sections;
+    for $model.evaluation-order.reverse -> $module {
+
+        my @inputs;
+        for $module.input -> $input (:$name, :$description, *%) {
```

Comment:
You're not using `$name` and `$description`; destructuring is not particularly cheap, so it's quite some waste if you don't use the result of it (and since we're pulling things out of the model object, I don't see much point in doing it for validation).

Reply:
Removed.